styleguide/_themes/global/scss/modifiers.scss

```diff
@@ -76,6 +76,18 @@ img { max-width: 100%; }
   margin: 0 auto;
 }
 
+// Horizontal rules
+hr {
```

Comment:
I would prefer we use hr tags for this but follow maintainable coding standards in our class naming like @WidgetsBurritos describes.